pyproject.toml

```diff
@@ -123,6 +125,7 @@ module = [
     "numpyro.contrib.hsgp.*",
     "numpyro.contrib.stochastic_support.*",
     "numpyro.diagnostics.*",
+    "numpyro.distributions.*",
```

Comment:
you can comment out this and add TODO link to https://github.com/pyro-ppl/numpyro/issues/2036